test-projects/debug-log/manifest.yml

```diff
@@ -0,0 +1,9 @@
+packages:
+  __APP_PACKAGE__:
+    actions:
+      worker:
+        function: 'worker.js'
+        runtime: 'nodejs:10'
```

Comment:
note: we should switch these to nodejs:14 soon given the runtime switch

Reply:
Or at least nodejs:12 (since it's in LTS now).

Reply:
These only run locally so I am not sure we have to update them. As long as the action test runner can find an image for it it should be ok.

In any case it should be a separate PR (there are many more existing test projects here).

Reply:
I was going to do a PR but noticed we have at least one [node12 test-project](https://github.com/adobe/aio-cli-plugin-asset-compute/tree/master/test-projects/node12) already.

All other tests are node 10. And the container version used is [hardcoded in the code](https://github.com/adobe/aio-cli-plugin-asset-compute/blob/master/src/lib/actionrunner.js#L37-L38). So it doesn't automatically change when Runtime drops node 10.